LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
overall -- let's follow the https://reuse.software/spec-3.3/  so there would be `LICENSES/` folder.